projects/docs-tools/src/modules/type-definitions/typedoc-adapter.service.ts

```diff
@@ -419,8 +419,13 @@ export class SkyDocsTypeDocAdapterService {
         },
       };
 
-      const tags = this.getCommentTags(indexSignature.comment);
-      this.applyCommentTagValues(definition, tags);
+      // New versions of typedoc do not seem to add the comment for index signatures - the else statement is a default based on the only index signature we had in September 2023 (SkySelectField).
+      if (indexSignature.comment) {
```

Comment:
Typedoc no longer outputs the descriptions for index signatures. I added this fallback to give a default (which matches the only instance of this that we have in our docs).